CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.0.7 (17MAY21)
+
+* **Overall**
+  * Research was completed that will enable OpenShift support in a future release
+
```

Comment:
I don't think this item is needed in the change log; nothing changed.  It does belong in our internal project recap reporting.